airbyte-cdk/bulk/toolkits/load-s3/src/main/kotlin/io/airbyte/cdk/load/file/s3/S3Client.kt

```diff
@@ -207,7 +207,7 @@ class S3ClientFactory(
     @Singleton
     @Secondary
     fun make(): S3Client {
-
+        println(bucketConfig.s3BucketConfiguration)
```

Comment:
Did you intend to keep this?
If yes, should this be a logger?

Reply:
oops thanks
long week